go/logic/migrator.go

```diff
@@ -518,19 +518,23 @@ func (this *Migrator) cutOver() (err error) {
 			}
 		}
 	}
-	if this.migrationContext.CutOverType == base.CutOverAtomic {
+	var cutOverFunc func() error
+	if !mysql.IsSmallerMinorVersion(this.migrationContext.ApplierMySQLVersion, "8.0.13") {
```

Comment:
@raidivyansh074 let's allow the user to decide if they want this style of cut-over, at least for now

Could you please add a new cut-over type the command line [here](https://github.com/github/gh-ost/blob/master/go/cmd/gh-ost/main.go#L98) and [here](https://github.com/github/gh-ost/blob/master/go/cmd/gh-ost/main.go#L263-L270)

The check for `mysql.IsSmallerMinorVersion` could move to be a check at the start of the new cut-over type only